server/sonar-web/src/main/js/apps/projects/store/utils.js

```diff
@@ -34,13 +34,15 @@ const getAsLevel = value => {
   return null;
 };
 
+// TODO Maybe use parseAsString form helpers/query
 const getAsString = value => {
   if (!value) {
     return null;
   }
   return value;
 };
 
+// TODO Maybe move it to helpers/query
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)